cfme/tests/configure/test_docs.py

```diff
@@ -84,8 +82,20 @@ def docs_info():
         return []
 
 
+@pytest.fixture(scope="session")
+def docs_content():
+    return {'monitoring_guide': 'monitoring, alerts, and reporting',
+            'user_guide': 'general configuration',
+            'vm_hosts_guide': 'provisioning virtual machines and hosts',
+            'lifecycle_and_automation_guide': 'methods available for automation',
+            'insight_guide': 'managing infrastructure and inventory',
+            'providers_guide': 'managing providers',
+            'scripting_actions_guide': 'scripting actions in cloudforms',
+            'control_guide': 'policies and profiles guide'}
+
+
 @pytest.mark.tier(2)
-@pytest.mark.meta(blockers=[1272618])
+@pytest.mark.ignore_stream("upstream")
```

Comment:
One last thing - since we mark all tests in this module with `ignore_stream('upstream')`, we should use `pytestmark = pytest.mark.ignore_stream("upstream")` at the top of this module instead. https://github.com/ManageIQ/integration_tests/blob/a97cf316a683009c9caa4db5c08d90bf5b1fa47a/cfme/tests/cloud/test_stack.py#L8